packages/codecs/src/anchor.ts

```diff
@@ -1,120 +1,69 @@
-import { type TypeOf, literal, optional, sparse, strict, string, union } from 'codeco'
-
+import { type, string, sparse, optional, literal, union, type TypeOf } from 'codeco'
 import { carAsUint8Array, cidAsString } from './ipld.js'
 import { streamIdAsString } from './stream.js'
+import { uint8ArrayAsBase64 } from './binary.js'
 
-/**
- * Describes all anchor statuses
- */
-export enum AnchorStatus {
-  NOT_REQUESTED = 0,
-  PENDING = 1,
-  PROCESSING = 2,
-  ANCHORED = 3,
-  FAILED = 4,
-  REPLACED = 5,
+export enum RequestStatusName {
+  PENDING = 'PENDING',
+  PROCESSING = 'PROCESSING',
+  COMPLETED = 'COMPLETED',
+  FAILED = 'FAILED',
+  READY = 'READY',
+  REPLACED = 'REPLACED',
 }
 
-export const AnchorProof = sparse(
-  {
-    chainId: string,
-    txHash: cidAsString,
-    root: cidAsString,
-    txType: optional(string),
-  },
-  'AnchorProof'
-)
-export type AnchorProof = TypeOf<typeof AnchorProof>
-
-export const AnchorCommit = strict(
-  {
-    id: cidAsString,
-    prev: cidAsString,
-    proof: cidAsString,
-    path: string,
-  },
-  'AnchorCommit'
-)
-export type AnchorCommit = TypeOf<typeof AnchorCommit>
-
-export const AnchorServicePending = strict(
+export const CommitPresentation = sparse(
   {
-    status: literal(AnchorStatus.PENDING),
-    streamId: streamIdAsString,
-    cid: cidAsString,
-    message: string,
+    content: optional(
+      sparse({ path: optional(string), prev: string, proof: optional(string) }, 'content')
```

Comment:
This whole `content` thing is a misnomer. There is no code that uses it now, and it is like that since quite a long time ago. Only `cid` is read. So, I just deleted `content`.